vendor/github.com/docker/go-metrics/gauge.go

```diff
@@ -14,7 +14,7 @@ type Gauge interface {
 	Set(float64)
 }
 
-// LabeledGauge describes a gauge the must have values populated before use.
+// LabeledGauge describes a gauge that must have values populated before use.
```

Comment:
@anshulpundir this change cannot be made here, since all files in the `vendor/` directory are pulled in from other projects (this is why the CI fails right now). You'll have to make this update in `github.com/docker/go-metrics` and then update the vendoring here.